instagram/src/App.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import './App.css';
+
+
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar';
+import PostsContainer from './components/PostContainer/PostContainer';
+
+class App extends Component {
+  constructor() {
+    console.log('Constructor running');
+    super();
+    this.state = {
+      dummyDataArray: [],
+      searchedPosts: []
+    };
+  }
+
```

Comment:
good job with the react lifestyle components here and the comments like on line 22.